tests/load/load_test.go

```diff
@@ -257,25 +263,16 @@ done:
 	gomega.Expect(err).Should(gomega.BeNil())
 	outf("{{blue}}avalanche HTTP RPCs URIs:{{/}} %q\n", uris)
 
-	for _, u := range uris {
-		rpcEP := fmt.Sprintf("%s/ext/bc/%s/rpc", u, blockchainID)
-		timestampvmRPCEps = append(timestampvmRPCEps, rpcEP)
-		outf("{{blue}}avalanche timestampvm RPC:{{/}} %q\n", rpcEP)
-	}
-
-	instances = make([]instance, len(uris))
-	for i := range uris {
-		u := uris[i] + fmt.Sprintf("/ext/bc/%s", blockchainID)
-		instances[i] = instance{
-			uri: u,
-			cli: client.New(u),
-		}
-	}
+	clientURIs = uris
 
-	defer outf("{{magenta}}logs dir:{{/}} %s\n", logsDir)
+	outf("{{magenta}}logs dir:{{/}} %s\n", logsDir)
```

Comment:
TODO: separate network start into helper, this is duplicated in e2e test as well